css/main.css

```diff
@@ -416,6 +417,41 @@ textarea {
     brightness(92%) contrast(82%);
 }
 
+#mobile-nav {
+  width: 100%;
+  position: absolute;
+  top: 0;
+  background: #3c3a39;
+  z-index: 100;
+  left: 0;
+  height: 80vh;
+  padding-top: 60px;
+}
+
+#mobile-nav li {
+  text-align: left;
+  border-bottom: 1px solid #777;
+  width: 85%;
+  margin: 0 auto;
+}
+
+#mobile-nav a {
+  color: whitesmoke;
+}
```

Comment:
- As well as the previous reviewer, I kindly suggest you to match the design of your mobile menu with the original [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) template, please check the difference: 

## Original: 

![image](https://user-images.githubusercontent.com/77038610/142751273-b27d684f-d94f-4581-b33d-dbd7e676c9e9.png)

## Yours:

![image](https://user-images.githubusercontent.com/77038610/142751281-17c71cd8-1bcf-4c2b-84a9-d3299030d40d.png)

- [ ] Please kindly change the `<a>` tag's `font-color` to `#FFF5E1`, and `font-weight` to `700`. You can get these and more details on Figma as shown in video below:

![chrome-capture (4)](https://user-images.githubusercontent.com/77038610/142751369-fad873d1-9556-4429-84bc-1984d79cbca4.gif)